test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/ImportTests.cs

```diff
@@ -164,6 +164,55 @@ private object ExecuteSql(string sql)
             return (checkLocation, id);
         }
 
+        [Fact]
+        public async Task GivenIncrementalLoad_1001ResourcesWithSameLastUpdatedAndSequenceRollOver()
+        {
+            if (!_fixture.IsUsingInProcTestServer)
+            {
+                return;
+            }
+
+            ExecuteSql("TRUNCATE TABLE EventLog");
+            ExecuteSql("DELETE FROM Resource");
```

Comment:
Is this going to affect other E2E tests since they run in parallel?

Reply:
Fixed.